### DISCUSSION TITLE
Barotauma:

### DISCUSSION THREAD

**Discussion by greggmon** [FakeFishGames/Barotrauma#13665] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When I use the device to make items and make one renewed welding tank it works but put in iso alchohol and empty tank again it crashes when trying to refill another tank or won't allow me to make another, crashed 1st time, 2nd time would n't allow me.

### Reproduction steps

now mutliplayer

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.2.8.0 (Winter Update hotfix 2)

### -

NA

### Which operating system did you encounter this bug on?

Windows

### Relevant error messages and crash reports

```shell
stops working
```

**Comment by Regalis11** [comment]
Insufficient information to address the issue
